specification/components/schemas/codeable/RemovalReasonExitCode.yaml

```diff
@@ -24,12 +24,12 @@ properties:
           description: A CodeSystem that identifies the reason a PDS record has been removed.
           example: SCT
           enum:
-            - DEA
-            - EMB
-            - SCT
-            - NIT
-            - TRA
-            - ORR
+            - DEA - Death
+            - EMB - Embarkation
+            - SCT - Transferred to Scotland
+            - NIT - Transferred to Northern Ireland
+            - TRA - Temporary resident not returned
+            - ORR - Other reason
```

Comment:
This change would imply/enforce that the string `"DEA - Death"` etc would be sent; whereas I believe the intention of your change is to provided an understanding of what DEA means.

Something like this might work ...
```
code:
    type: string
    description: A CodeSystem that identifies the reason a PDS record has been removed.
    example: SCT
    oneOf: 
        - const: "DEA"
          description: "Death" 
        - const: "EMB" 
          description:  Embarkation
        ...
```

Reply:
That makes sense, thank you! I'll get that fixed.